nservicebus/upgrades/sqlserver_2to3.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Upgrade from Version 5 to Version 6
+summary: Instructions on how to upgrade from NServiceBus Versions 5 to 6
+tags:
+ - upgrade
+ - migration
+related:
+- nservicebus/sagas/concurrency
+---
+
+
+## SQL Server Transport
+
+### Transactions
+Transaction settings are now handled in the transport level configuration. For details refer to [Transaction configuraton API](/nservicebus/upgrades/5to6.md#transaction-configuration-API) section.
+
+The native transaction support has been split into two different levels: `ReceiveOnly` and `SendAtomicWithReceive`. SQL Server Transport supports both of them. `SendAtomicWithReceive` is equivalent to disabling distributed transactions in Version 5. For more details refer to [Transaction configuraton API](/nservicebus/upgrades/5to6.md#transaction-configuration-API) section and [Transaction support](/nservicebus/messaging/transactions.md) page.
```

Comment:
Again, I wanted to avoid repetition. Do you think it's worth duplicating snippets in that case?

> I was using the previous version and was disabling DTC, how do I do that in the new version. How would I have to change my code.

> `SendAtomicWithReceive` is equivalent to disabling distributed transactions in Version 5

Is the problem that the statement above is not precise enough or that the formatting could be better?